test/fail_compilation/fail17722b.d

```diff
@@ -0,0 +1,13 @@
+/* TEST_OUTPUT:
```

Comment:
Please merge these two test files into one. It'll consume half the resources of the autotester that way.